PowerShell/ScubaGear/Rego/AADConfig.rego

```diff
@@ -298,15 +305,42 @@ tests contains {
 # MS.AAD.3.5v1
 #--
 
-# At this time we are unable to test for SMS/Voice settings due to lack of API to validate
-# Awaiting API changes and feature updates from Microsoft for automated checking
+GoodAuthenticationMethodConfigurations contains {
+    "Id": Configuration.Id,
+    "State": Configuration.State
+} if {
+    some Item in input.authentication_method
+    some Configuration in Item.AuthenticationMethodConfigurations
+    Configuration.Id in ["Sms", "Voice", "Email"]
+    Configuration.State == "disabled"
+}
+
 tests contains {
-    "PolicyId": "MS.AAD.3.5v1",
-    "Criticality": "Shall/Not-Implemented",
-    "Commandlet": [],
+    "PolicyId": PolicyId,
+    "Criticality": "Shall",
+    "Commandlet": ["Get-MgBetaPolicyAuthenticationMethodPolicy"],
     "ActualValue": [],
-    "ReportDetails": NotCheckedDetails("MS.AAD.3.5v1"),
+    "ReportDetails": CheckedSkippedDetails("MS.AAD.3.4v1", Reason),
```

Comment:
```suggestion
    "ReportDetails": CheckedSkippedDetails("MS.AAD.3.5v1", Reason),
```
I thought this was 3.5?

Reply:
I am fairly sure there is only one policy --  but all things generally get wrapped with an array (not 100% sure why that was decided).

Reply:
The message refers to 3.5 is skipped because 3.4 is not satisfied.